%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
Can't connect to pigpio at localhost(8888)

Did you start the pigpio daemon? E.g. sudo pigpiod

Did you specify the correct Pi host/port in the environment
variables PIGPIO_ADDR/PIGPIO_PORT?
E.g. export PIGPIO_ADDR=soft, export PIGPIO_PORT=8888

Did you specify the correct Pi host/port in the
pigpio.pi() function? E.g. pigpio.pi('soft', 8888)
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 * Serving Flask app "command" (lazy loading)
 * Environment: production
   WARNING: This is a development server. Do not use it in a production deployment.
   Use a production WSGI server instead.
 * Debug mode: off
 * Running on http://0.0.0.0:9000/ (Press CTRL+C to quit)
127.0.0.1 - - [22/May/2023 16:39:57] "[37mGET /command/1/400/14000 HTTP/1.1[0m" 200 -
127.0.0.1 - - [22/May/2023 16:41:07] "[37mGET /command/1/400/14000 HTTP/1.1[0m" 200 -
127.0.0.1 - - [22/May/2023 16:45:02] "[37mGET /command/2/300/14000 HTTP/1.1[0m" 200 -
